src/matter/cluster/OnOffCluster.ts

```diff
@@ -4,12 +4,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { BooleanT } from "../../codec/TlvObjectCodec";
+import { Bit, BooleanT } from "../../codec/TlvObjectCodec";
 import { Attribute, Cluster, Command, NoArgumentsT, NoResponseT } from "./Cluster";
 
 export const OnOffCluster = Cluster({
     id: 0x06,
     name: "On/Off",
+    revision: 1,
+    features: {
+        lightingLevelControl: Bit(0),
```

Comment:
Whats the value if not defined in "Device.ts"? will it be 0?

Reply:
Features have to be defined in Device.ts as either true or false.
If this becomes too annoying, I can modify the code to have features optional with false as the default.

Reply:
All fine but then this is missing or?!

Reply:
in device.ts in the defined onoff cluster there is nothing set for that bit ... but typescript accepts that ...

Reply:
You are correct, I forget one piece... Working late at night is not always the best. Fixed now.